components/camel-milo/src/main/java/org/apache/camel/component/milo/client/MiloClientProducer.java

```diff
@@ -35,6 +37,8 @@
 
     private final boolean defaultAwaitWrites;
 
+    private static final String HEADER_NODE_IDS = "opcua.nodeIds";
```

Comment:
Why is the header key with this value? - what is opcua

Reply:
opcua is the protocol underneath. But i will remove the prefix. The existing headers has also no prefix.

Reply:
changed it to CamelMilo.nodeIds